### DISCUSSION TITLE
Creating a Svelte port of Geist

### DISCUSSION THREAD

**Discussion by Zk2u** [zeit-ui/react#764] [Ideas]
Is there any interest for a port of Geist to be created in Svelte? Or any plans?

**Comment by OStirbu** [comment]
Would love that 🖤

**Comment by unix** [comment]
Honestly, I don't have much experience with svelte and don't know how much it would cost to migrate or be compatible~ 🤣

**Comment by hammadmajid** [comment]
2 years later still no port? Would love use this design system in Svelte
